package.json

```diff
@@ -20,6 +21,17 @@
   },
   "homepage": "https://github.com/FACG6/AMiDEAST#readme",
   "devDependencies": {
-    "concurrently": "^4.1.0"
+    "concurrently": "^4.1.0",
+    "nodemon": "^1.18.11",
+    "supertest": "^4.0.2",
+    "tap-spec": "^5.0.0",
+    "tape": "^4.10.1"
+  },
+  "dependencies": {
+    "bcrypt": "^3.0.6",
+    "dotenv": "^7.0.0",
+    "i": "^0.3.6",
```

Comment:
what's the benefit of installing  ``` bcrypt``` , ```jsonwebtoken```, ```dotenv```, ``` i``` and ```npm```  packages  when you set up the Travis?!!
And what is ```i``` package ?

Reply:
done